openstudiocore/src/airflow/contam/ForwardTranslator.cpp

```diff
@@ -434,10 +434,18 @@ boost::optional<contam::IndexModel> ForwardTranslator::translateModel(model::Mod
   m_logSink.setThreadId(QThread::currentThread());
   m_logSink.resetStringStream();
 
-  m_prjModel.read(std::string(":/templates/template.prj"));
-  if (!m_prjModel.valid()) {
-    return boost::none;
+  {
+    QFile f(":/templates/template.prj");
```

Comment:
@lefticus What is going on here? one would assume you are on the pursuit of getting rid of QFile, but here obviously you can't since a q resource is involved. Are you trying to stage for the future in some way i don't entirely understand.